src/components/Button/PlayButton.js

```diff
@@ -15,15 +26,32 @@ const StyledPlayButton = styled.div`
     position: relative;
     left: 3px;
   }
-  ${props => props.hover ? 'opacity: 0;' : ''},
+  ${props => props.hover ? 'opacity: 0;' : ''}
+  ${props => 
+    props.type === 'lesson' ||
+    (props.type === 'course' && (props.expanded === false || props.expanded === 'horizontal'))
+     ? `${centerButtonRules}`
```

Comment:
if it's a `LessonCard` or a "regular" `CourseCard` or a Horizontal `CourseCard`, apply the CSS rules in `centerButtonRules`